### ISSUE TITLE
Key hash tag for PECL Redis

### ISSUE THREAD

**Issue by szepeviktor** [laravel/docs#4406]
Hello!

Today we've found out the [key hash tag](https://github.com/laravel/docs/blob/5.6/queues.md#redis) is also necessary for queues with Redis driver implemented by PECL Redis extension (`'client' => 'phpredis',`) even for single (local) Redis server.

Could you rephrase that paragraph?

**Comment by laurencei** [comment]
Hi - are you able to explain a bit further? Can you give an example of what you think the paragraph should look like?

**Comment by szepeviktor** [comment]
Hi @laurencei !
The one thing I'm sure of is our queue workers couldn't get user details without setting `{default}` as queue name.
When switching to `predis` it worked again.
After some googling I've concluded the above situation.

As for rephrasing the docs I would say *Redis Cluster and single Redis using the phpredis extension*.
I hope it helps, I am not a native speaker.

**Comment by driesvints** [comment]
I've not used the phpredis extension myself but if you have and feel that the docs can be more clear about this feel free to send in a PR and update the parts where it needs to be more clear.

**Comment by szepeviktor** [comment]
> I've not used the phpredis extension myself

It is written in C: tiny and fast.
